source/core/Math.cpp

```diff
@@ -12,6 +12,10 @@ SDG
  */
 
 #include <cstdlib>
+
+#ifdef _MSC_VER
```

Comment:
How about the following in BuildConfig.h (and use kVireoCompiler_MSVC here)?
#if defined(_MSC_VER)
	#define kVireoCompiler_MSVC 1
#endif
I think we want to distinguish OS vs Compiler.